src/code/entities/bank3.asm

```diff
@@ -4815,9 +4817,9 @@ jr_003_6422:
     dec  e                                        ; $6446: $1D
     ld   a, e                                     ; $6447: $7B
     cp   $FF                                      ; $6448: $FE $FF
-    jr   nz, jr_003_6422                          ; $644A: $20 $D6
+    jr   nz, .movePickupHigher                    ; $644A: $20 $D6
 
-PickEntity32::
+.return::
```

Comment:
Nitpick: I think you don't need the `::` here to make the `MovePickupInTheAir.return` available.